canonical/ubuntu.com
# WD-11695 - redirect users to different takeovers

## Done

Added switching between 2 takeover types for different user groups. PUT SWITCH TO FALSE BEFORE MERGE

## QA

- [demo link](https://ubuntu-com-13990.demos.haus/)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Find control_or_variant cookie, delete it a couple times to see takeover change between options:
(may be different content, mind layout only)
<img width="1199" alt="Screenshot 2024-06-27 at 11 55 42" src="https://github.com/canonical/ubuntu.com/assets/54859248/a63ac2f8-b1cd-44f3-bba0-8198ca0be18e">

<img width="969" alt="Screenshot 2024-06-27 at 10 40 23" src="https://github.com/canonical/ubuntu.com/assets/54859248/111471ce-1a14-4d5e-b06b-79be5bd43f09">

- Check that the takeover changes correctly according to the user type

## Issue / Card
[WD-11695](https://warthogs.atlassian.net/browse/WD-11695)

[WD-11695]: https://warthogs.atlassian.net/browse/WD-11695?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
